### ISSUE TITLE
Positive hair pull test

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11464]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 28010890 Hair loss disorders are ideally monitored using quick, noninvasive clinical examinations, such as the **hair pull test**. To perform this test, the clinician selects 50 to 60 hairs and holds the bundle close to the scalp between the thumb, index finger, and long finger. The clinician then firmly pulls on the bundle using slow traction as the fingers slide down the hair shaft, avoiding a fast and forceful tug. The hair pulls are performed at the vertex, parietal areas, and the occipital area of the scalp. Next, the pulled hairs are counted. Any broken hairs that were extracted from the bundle during the pull are discarded. If more than 10% of the hairs in each bundle are removed from a scalp area, the hair pull test is considered positive. If fewer than 10% are removed, then the hair loss can usually be attributed to normal shedding. If a test is positive in more than 1 scalp region, the clinician must consider telogen or anagen effluvium. Other hair disorders (ie, alopecia areata) may only have a positive hair pull test in the affected area.
The hair pull test is ideally used for monitoring the advancing edge of alopecia areata, acute cases of telogen effluvium, anagen effluvium, and loose
anagen syndrome. 

**Parent term (use hpo.jax.org/app)**
Abnormality of the scalp hair  HP:0100037

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM104000 ALOPECIA AREATA

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001481
- label: Positive hair pull test
- parents: [Abnormality of the scalp hair (HP:0100037)](https://hpo.jax.org/app/browse/term/HP:0100037)
- pmid: PMID:28010890
- def: An abnormal result of the hair pull test, in which the clinician selects 50 to 60 hairs and holds the bundle close to the scalp between the thumb, index finger, and long finger. The clinician then firmly pulls on the bundle using slow traction as the fingers slide down the hair shaft, avoiding a fast and forceful tug. The hair pulls are performed at the vertex, parietal areas, and the occipital area of the scalp. Next, the pulled hairs are counted. Any broken hairs that were extracted from the bundle during the pull are discarded. If more than 10 percent of the hairs in each bundle are removed from a scalp area, the hair pull test is considered positive.
- comment: If fewer than 10% are removed, then the hair loss can usually be attributed to normal shedding. If a test is positive in more than one scalp region, the clinician must consider telogen or anagen effluvium. Other hair disorders (ie, alopecia areata) may only have a positive hair pull test in the affected area. The hair pull test is ideally used for monitoring the advancing edge of alopecia areata, acute cases of telogen effluvium, anagen effluvium, and loose anagen syndrome.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11464
- synonyms:
